QUEST.tsv

```diff
@@ -1499,155 +1499,155 @@ QUEST_20150323_001498	Okay, that'll be only 1,000 silver coins then.
-QUEST_20150323_001583	You will enter the special room when you get all five keys of Mayvern. {nl}In there, you wil be tested to see if you're eligible for the revelation. 
+QUEST_20150323_001574	$You must have a lot of questions. Ask, Revelator…
+QUEST_20150323_001575	$Vakarine is fighting Valtrus in the Demon Prison across the portal. {nl}She won't be able hold it for much longer unless someone helps.
+QUEST_20150323_001576	$Men under the command of the Baron have been here though they weren't able to make it back. {nl}I watched everything with the wizard's eyes. {nl}Even the screams of Vakarine and her Cupols.
+QUEST_20150323_001577	$That depends on whether or not you have the will to save the Goddess. {nl}We can take time to talk about it. 
+QUEST_20150323_001578	$The limbs of Nuaelle were all cut off. {nl}Now go find Aldona. {nl}She will take you and Houberg to Nuaelle's territory.
+QUEST_20150323_001579	$We're entering Nuaelle's area now. {nl}Get ready. 
+QUEST_20150323_001580	$Once Nuaelle is gone, we'll be able to fight the demons easily even with only the Cupols. 
+QUEST_20150323_001581	$Our biggest problem aside from the corruption breach was Nuaelle. {nl}Now we can focus on the former. {nl}Go to Prison District 3. The Goddess awaits your support.
+QUEST_20150323_001582	$Keep your guard up. {nl}It's the secret to the last key but the demons are still watching.
+QUEST_20150323_001583	$You will enter the special room once you get all five keys of Maven. {nl}In there, you will be tested to see if you're eligible for the revelation. 
 QUEST_20150323_001584	The number of candles is important but for this secret, you must remember the order of the secret too. {nl}It is similar to the first secret.
 QUEST_20150323_001585	Device is operating
-QUEST_20150323_001586	A few addicted farmers assaulted the people. 
+QUEST_20150323_001586	$A few manic farmers assaulted the people. 
 QUEST_20150323_001587	Guards are working hard to protect the farm. {nl}Help them. 
-QUEST_20150323_001588	I thank you again in the name of all fate. {nl}From here, I will follow the stars to find Goddess Ausrine.{nl}
-QUEST_20150323_001589	The seal of Uskis Arable Land is now safe with your help. {nl}Other Revelators lost their consiousness due to the evil force coming from the Spring Light Woods.{nl}
-QUEST_20150323_001590	This Cathedral was built 660 years ago. {nl}It was built as the most glorious church under the guide of archbishop Silvestro Mayvern.{nl}
-QUEST_20150323_001591	It's like I can still hear it. {nl}Prayers of the priests filled the kingdom. {nl}There were endless march of Pilgrims coming for the grace of the Goddess.{nl}
-QUEST_20150323_001592	It was like that until four years ago. {nl}Before the curse surrounded the cathedral on Divine Day…
+QUEST_20150323_001588	$I thank you again in the name of fate. {nl}From here on, I will climb the stars to find Goddess Ausrine.
```

Comment:
"follow"ing the stars is something an astrologist would do, in order to discern fate.  So the original makes sense, I wouldn't understand "climb the stars".